packages/core/src/components/chip/chip.tsx

```diff
@@ -71,16 +71,27 @@ export class Chip {
 
   /**
    * Fire event if close button is clicked
+   *
+   *  @deprecated
    */
   @Event() close: EventEmitter;
 
+  /**
+   * Fire event if close button is clicked
+   */
+  @Event() closeChip: EventEmitter;
+
   private getCloseButton() {
     return (
       <div class="close-button-container">
         <button
           type="button"
           class="btn btn-invisible-secondary btn-icon btn-oval close-button"
-          onClick={(event) => this.close.emit(event)}
+          onClick={(event) => {
+            this.close.emit(event);
+            this.closeChip.emit(event);
+            event.stopImmediatePropagation();
```

Comment:
Is there any specific reason behinde the stop propagation?

Reply:
The reason is, when clicked on the close button, it is also emiting the main button event. 
Did a new commit where i changed stopImmediatePropagation to stopPropagation.